### PR TITLE
Added Onemanhua source

### PR DESCRIPTION
**Pull request by lmk1988**
#1369

### CODE REVIEW

File: src/zh/onemanhua/src/eu/kanade/tachiyomi/extension/zh/onemanhua/Onemanhua.kt [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+        return Headers.Builder()
+            .add("User-Agent", "Tachiyomi " + System.getProperty("http.agent"))
+    }
+
+    // Don't spam the server with your fast fingers
+    private val rateLimitInterceptor = RateLimitInterceptor(4)
+
+    override val client: OkHttpClient = network.client.newBuilder()
+        .addNetworkInterceptor(rateLimitInterceptor)
+        .build()
```
Comment (arkon) [inline src/zh/onemanhua/src/eu/kanade/tachiyomi/extension/zh/onemanhua/Onemanhua.kt:40]: Is this necessary, or just copied from other extensions?

Comment (lmk1988) [reply src/zh/onemanhua/src/eu/kanade/tachiyomi/extension/zh/onemanhua/Onemanhua.kt:40]: Not necessary. Looks like good practice from other extensions, hence followed

Comment (arkon) [reply src/zh/onemanhua/src/eu/kanade/tachiyomi/extension/zh/onemanhua/Onemanhua.kt:40]: Let's remove it if not necessary. We usually only send user agent strings like that if requested. Similarly, rate limiting is useful only if the website tends to ban people.

Comment (lmk1988) [reply src/zh/onemanhua/src/eu/kanade/tachiyomi/extension/zh/onemanhua/Onemanhua.kt]: Removed headers and rate limiting

**Comment by pavan3999** [comment]
merged? i didn't find this extension in list

**Comment by arkon** [comment]
Not sure why it didn't trigger a build. Manually kicked one off: https://travis-ci.org/github/inorichi/tachiyomi-extensions/builds/672520271

**Pull request merged**